src/include/netloc_util.h

```diff
@@ -25,6 +26,13 @@ typedef struct {
              * the network */
             int *node_levels;
         } tree;
+        struct {
+            int dimension;
+            int *geometry;
+            /* Flat index of the node the current
+             * rank is mapped to */
+            int current_node_coordinates;
```

Comment:
Can you use the name `node_idx` instead of `current_node_coordinates`, to make this less confusing?

Reply:
Done